lib/reporters/html/partials/aggregations.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if folder}}
```

Comment:
@malaubay Can you open a new issue for your query? It'll be easier for other people who have the same question to find the answer :)